### DISCUSSION TITLE
How to list all items or start fresh?

### DISCUSSION THREAD

**Discussion by davidc360** [FelixKratz/SketchyBar#136] [Q&A] [answered]
Hi all, I started with the base template and I want to start fresh and build my own. Is there a way to list all the current items so I can remove them? And is there a way to start with a fresh template? Thanks

**Comment by FelixKratz** [comment, marked as answer]
Have a look at the config file in `~/.config/sketchybar/` there you can set everything up to your liking

**Reply by davidc360** [reply]
thank you!
